src/applications/personalization/dashboard-2/components/apply-for-benefits/ApplicationInProgress.unit.spec.jsx

```diff
@@ -1,38 +1,81 @@
 import React from 'react';
+import sinon from 'sinon';
 import { expect } from 'chai';
 import { render } from '@testing-library/react';
 
 import ApplicationInProgress from './ApplicationInProgress';
 
 describe('ApplicationInProgress component', () => {
-  const props = {
-    continueUrl: 'application-url/',
-    expirationDate: 'Jan 1, 2020',
-    formId: '1234',
-    formTitle: 'Form Title',
-    lastOpenedDate: 'Jan 1, 2019',
-    presentableFormId: 'Form 1234',
+  const defaultProps = () => {
+    return {
+      continueUrl: 'application-url/resume',
+      formId: '1234',
+      formTitle: 'form title',
+      lastOpenedDate: 'Jan 1, 2019',
+      presentableFormId: 'Form 1234',
+      removeForm: () => {},
+      startNewApplicationUrl: 'application-url',
+    };
   };
-  let view;
-  beforeEach(() => {
-    view = render(<ApplicationInProgress {...props} />);
+  const oneWeekInMs = 7 * 24 * 60 * 60 * 1000;
```

Comment:
Ha, I'm wondering if we should move these computed dates into a higher level folder here soon. We're using them a lot in our tests now 😆

Reply:
Yeah. This should really live in platform, IMO.